tests/conftest.py

```diff
@@ -50,3 +52,10 @@ def tls_ca_certificate_private_key_encrypted_path(tls_certificate_authority):
 def tls_certificate_pem_path(tls_certificate):
     with tls_certificate.private_key_and_cert_chain_pem.tempfile() as cert_pem:
         yield cert_pem
+
+
+@pytest.fixture
+def ssl_ctx(tls_certificate):
+    ssl_ctx = ssl.SSLContext(ssl.PROTOCOL_SSLv23)
```

Comment:
I don't think there's actually a need for specifying anything, is there?

```suggestion
    ssl_ctx = ssl.SSLContext()
```

Reply:
it seems like it doenst